Language/Classes/Instance_Methods/covariant_A04_t01.dart

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion For each parameter p of m where covariant is present, it is a
+/// compile-time error if there exists a direct or indirect superinterface of C
+/// which has an accessible method signature m′′ with the same name as m, such
+/// that m′′ has a parameter p′′ that corresponds to p, unless the type of p is
+/// a subtype or a supertype of the type of p′′
+///
+/// @description Checks that tearing-off a method with a covariant-by-class
+/// parameter strips a `covariant` keyword
```

Comment:
I'm not sure I understand this description. ;-)

I can't see any occurrences of `covariant` in this test, so there's nothing to strip, and if we're talking about function types then they don't ever contain that keyword, so it doesn't make sense to say that we're stripping it off of a function type.

I guess the test is about the static type of a tear-off of an instance method where a parameter is covariant-by-class?